### PR TITLE
Introduce new command "--active-stylus-delta" to capture the deltas of Active Stylus

### PR DESCRIPTION
**Pull request by kdunning**
Updated from https://github.com/atmel-maxtouch/mxt-app/pull/38, removing static function mxt_debug_insert_data_ast, which was preventing compilation.

### CODE REVIEW

**Comment by kdunning** [comment]
@drxaero, whilst trying to merge https://github.com/atmel-maxtouch/mxt-app/pull/38 into the latest master branch, I found that it was failing to build due to an unused static function (mxt_debug_insert_data_ast). Having looked through your changes, it appears that it is safe to remove this, and subsequent builds now work.

Could you possibly test this latest pull request to ensure the same performance as your previous pull request please?

Thanks.

**Comment by drxaero** [comment]
@kdunning,
After testing the latest commit  c7b5ecb with removed `mxt_debug_insert_data_ast()`, I confirm it's the same performance as my previous pull request.
So, please feel free to remove `mxt_debug_insert_data_ast()` from my previous pull request.

Thanks.

File: src/mxt-app/diagnostic_data.c [hunk 1]
```diff
@@ -214,6 +217,53 @@ static int mxt_generate_hawkeye_header(struct t37_ctx *ctx)
           return MXT_ERROR_IO;
       }
     }
+  } else if (ctx->active_stylus) {
+    for (pass = 0; pass < ctx->passes; pass++) {
```
Comment (kdunning) [inline src/mxt-app/diagnostic_data.c]: * next to variable

File: src/mxt-app/diagnostic_data.c [hunk 2]
```diff
@@ -214,6 +217,53 @@ static int mxt_generate_hawkeye_header(struct t37_ctx *ctx)
...
+      const char * mode;
```
Comment (kdunning) [inline src/mxt-app/diagnostic_data.c]: Pointer `*` next to variable

**Pull request merged**